src/esm_tests/cli.py

```diff
@@ -142,12 +151,17 @@ def main():
         sys.exit(1)
 
     # Get user info for testing
-    info = user_config(info)
+    user_config(info)
 
     # User-specific Info to remove from the files ``last_tested`` files
+    info["mnt"] = ""
+    pwd = os.getcwd()
+    if pwd.startswith("/mnt/lustre"):
+        info["mnt"] = "/".join(pwd.split("/")[:3])
```

Comment:
As above, this works, but looks confusing. Probably it would be nice to have something in the `helpers.py` which can mutate paths. I have no problem there including computer-specific functions. As an example:
```python
def clean_computer_specific_paths(path, computers=None):
    if computers is None:
        # Or some sensible default:
        computers = ["mistral", "levante"]:
    # ....something that grabs the computer config yaml .....
    for computer_config in computer_configs:
        for remove_prefix in computer_config["allowed_removes"]:
            path = path.replace(remove_prefix, "")
    return path



```

Reply:
The user and computer specific parts have been generalized and included into a method.